package.json

```diff
@@ -93,6 +92,7 @@
     "dirty-chai": "^2.0.1",
     "go-ipfs-dep": "0.4.13",
     "ipfs": "^0.27.5",
+    "ipfs-api": "^17.3.0",
```

Comment:
@dryajov only realized after that ipfs-api is now a devDep. Does this mean that folks have to pass ipfs-api everytime they want to spawn a node?

Reply:
It continues to be a prod dep https://github.com/ipfs/js-ipfsd-ctl/blob/master/src/daemon-node.js#L5

Reply:
What was the rationale behind moving it?